var/spack/repos/builtin/packages/brion/package.py

```diff
@@ -37,14 +37,10 @@ class Brion(CMakePackage):
     # TODO: bzip2 is a dependency of boost. Needed here because of linking
     # issue (libboost_iostreams.so.1.68.0 not finding libbz2.so)
     depends_on('bzip2')
-    depends_on('lunchbox')
-    depends_on('vmmlib')
```

Comment:
:clap: 

Although, wouldn't you want to pin that for certain versions? Or are we just dropping support outright?

Reply:
yes, actually we will keep previous release, so will keep them pinned to 3.1.0